driverdataaggregation/README.md

```diff
@@ -0,0 +1,407 @@
+# Data Aggregation Scaling Tests
+
+SmartSim-Scaling currently offers three data aggregation test versions:
+
+ 1. Data Aggregation Standard               (C++ Client and SmartRedis Orchestrator)
+ 2. Data Aggregation Standard Python        (Python Client and SmartRedis Orchestrator)
+ 3. Data Aggregation Standard File System   (Python Client and File System)
```

Comment:
This reads like we've implemented a file system in Python. I think we need to tweak the wording here

Reply:
I think I resolved this?